iothub/device/src/Transport/Mqtt/MqttTransportHandler.cs

```diff
@@ -77,13 +80,15 @@ internal sealed class MqttTransportHandler : TransportHandler, IMqttIotHubEventH
 
         // Topic names for receiving cloud-to-device messages.
         private const string DeviceBoundMessagesTopicFilter = "devices/{0}/messages/devicebound/#";
+
```

Comment:
why a new line here? the comment is relevant for both of these variables

Reply:
VS does it. It thinks a comment should only apply to the thing below it. The only way to fix that is to put a newline after the comment.